lib/modules/deployment/contract_deployer.js

```diff
@@ -257,7 +257,8 @@ class ContractDeployer {
             if (err) {
               return next(err);
             }
-            contract.gas = gasValue;
+            let increase_per = 1 + (Math.random() / 10.0)
```

Comment:
Can we comment on the intent of doing this?

Reply:
needs semicolon